index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html ng-app>
+<title>Angular Flashcards</title>
+<head>
+  <meta charset="UTF-8">
+  <title>Document</title>
+  <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.14/angular.min.js"></script>
+</head>
+<body>
+{{"hello" + " " + "world"}}
```

Comment:
This is a practice comment to show how to make comments when doing a code review.